reconcile/utils/terraform_client.py

```diff
@@ -142,7 +142,8 @@ def init_specs(self):
             TerraformSpec(name=name, working_dir=wd)
             for name, wd in self.working_dirs.items()
         ]
-        threaded.run(self.terraform_init, self.specs, self.thread_pool_size)
+        for spec in self.specs:
+            self.terraform_init(spec)
```

Comment:
would it make sense to have a dedicated working dir for each `spec`, avoiding any conflict?

Reply:
currently each working dir should already be uniq, generated here https://github.com/app-sre/qontract-reconcile/blob/1b57e81edae9e5d8dd1add675218b744e4fa3bcd/reconcile/utils/terrascript_aws_client.py#L4009